packages/acceptance-tests/pkg-tests-specs/sources/commands/__snapshots__/config.test.js.snap

```diff
@@ -0,0 +1,577 @@
+pnpPath                  <default>                                                                                                         'WORKSPACE_ROOT/.pnp.js'
+pnpShebang               <default>                                                                                                         '#!/usr/bin/env node'
+pnpUnpluggedFolder       <default>                                                                                                         'WORKSPACE_ROOT/.yarn/unplugged'
+preferInteractive        <default>                                                                                                         false
+rcFilename               <default>                                                                                                         '.spec-yarnrc'
+virtualFolder            <default>                                                                                                         'WORKSPACE_ROOT/.yarn/virtual'
+yarnPath                 <default>                                                                                                         null
+",
+}
+`;
+
+exports[`Commands config test (folder with rcfile and rc in parent / without flags) 1`] = `
+Object {
+  "code": 0,
+  "stderr": "",
+  "stdout": "bstatePath               'WORKSPACE_ROOT/.yarn/build-state.yml'
+cacheFolder              'WORKSPACE_ROOT/.yarn/cache'
+checksumBehavior         'throw'
+defaultLanguageName      'node'
+defaultProtocol          'npm:'
+enableAbsoluteVirtuals   true
+enableColors             false
+enableGlobalCache        false
+enableNetwork            true
+enableScripts            true
+enableTimers             false
+frozenInstalls           false
+globalFolder             'WORKSPACE_ROOT/.berry/global'
+httpProxy                null
+httpsProxy               null
+ignorePath               false
+initLicense              null
+initScope                'berry-test'
+initVersion              null
+lastUpdateCheck          '1555784893958'
+lockfilePath             'WORKSPACE_ROOT/yarn.lock'
+nodeGypMagicLocator      'node-gyp'
+npmRegistryServer        'http://yarn.test.registry'
+pnpDataPath              'WORKSPACE_ROOT/.pnp.data.json'
+pnpEnableInlining        true
+pnpIgnorePattern         null
+pnpPath                  'WORKSPACE_ROOT/.pnp.js'
+pnpShebang               '#!/usr/bin/env node'
+pnpUnpluggedFolder       'WORKSPACE_ROOT/.yarn/unplugged'
+preferInteractive        false
+rcFilename               '.spec-yarnrc'
+virtualFolder            'WORKSPACE_ROOT/.yarn/virtual'
+yarnPath                 null
+",
+}
+`;
+
+exports[`Commands config test (folder with rcfile without trailing newline / as json) 1`] = `
+Object {
+  "code": 1,
+  "stderr": "",
+  "stdout": "Error: Expected \\":\\", \\"\\\\n\\", \\"\\\\r\\", \\"\\\\r\\\\n\\", [^\\\\r\\\\n\\\\t :,], or blank space but end of input found (line 1, column 22).
```

Comment:
Reason for this error: the yarnrc file doesn't contain any lines (there's content but no newline)

While I'm firmly in the "every file _must_ end with EOL" camp, I know this is going to bite people. I'm not sure where to modify this though... I could add a newline here:

https://github.com/yarnpkg/berry/blob/f3f38740b16d49c81b42c47e44acc8f1c193c09b/packages/berry-core/sources/Configuration.ts#L440

but maybe it's better to make this part of the `@berry/parsers` package instead? i.e. add a newline here:

https://github.com/yarnpkg/berry/blob/f3f38740b16d49c81b42c47e44acc8f1c193c09b/packages/berry-parsers/sources/syml.ts#L89

Reply:
Yep, adding a virtual newline in `syml.ts` sounds like reasonable 👍